scripts/m0_niah.py

```
@@ -136,24 +147,34 @@ def main() -> int:
             f"in the passage? Answer with the exact value as written."
         )
         try:
-            ans = chat_complete(args.base_url, args.model, system, question,
-                                max_tokens=768)
+            ans = chat_complete(
+                args.base_url, args.model, system, question, max_tokens=768
+            )
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)